Freshli.Test/Acceptance.RequirementsTxtPyspider.approved.txt

```diff
@@ -57,7 +57,7 @@ results[55] = { Date: 2018-10-01T00:00:00, LibYear: { _packagesValues: [ { Name:
+results[59] = { Date: 2019-02-01T00:00:00, LibYear: { _packagesValues: [ { Name: "Flask", Version: "1.0.2", PublishedAt: 2018-05-02T14:26:26, Value: 0 }, { Name: "Jinja2", Version: "2.10", PublishedAt: 2017-11-08T20:13:42, Value: 0 }, { Name: "chardet", Version: "3.0.4", PublishedAt: 2017-06-08T14:34:33, Value: 0 }, { Name: "cssselect", Version: "1.0.3", PublishedAt: 2017-12-27T16:07:20, Value: 0 }, { Name: "lxml", Version: "4.3.0", PublishedAt: 2019-01-04T16:38:39, Value: 0 }, { Name: "pycurl", Version: "7.43.0", PublishedAt: 2016-02-05T06:09:27, Value: 0 }, { Name: "pyquery", Version: "1.4.0", PublishedAt: 2018-01-11T19:50:58, Value: 0 }, { Name: "requests", Version: "2.21.0", PublishedAt: 2018-12-10T15:40:08, Value: 0 }, { Name: "tornado", Version: "5.1.1", PublishedAt: 2018-09-16T19:22:58, Value: 0.6932902714358194 }, { Name: "mysql-connector-python", Version: "8.0.14", PublishedAt: 2019-01-21T07:56:51, Value: 0 }, { Name: "pika", Version: "0.13.0", PublishedAt: 2019-01-17T17:35:15, Value: 0 }, { Name: "pymongo", Version: "3.7.2", PublishedAt: 2018-10-10T19:19:51, Value: 0 }, { Name: "unittest2", Version: "1.1.0", PublishedAt: 2015-06-30T06:48:25, Value: 0 }, { Name: "Flask-Login", Version: "0.4.1", PublishedAt: 2017-12-02T02:35:30, Value: 0 }, { Name: "u-msgpack-python", Version: "2.5.0", PublishedAt: 2018-04-01T01:04:34, Value: 0 }, { Name: "click", Version: "7.0", PublishedAt: 2018-09-25T20:57:05, Value: 0 }, { Name: "SQLAlchemy", Version: "1.2.17", PublishedAt: 2019-01-26T02:31:42, Value: 0 }, { Name: "six", Version: "1.12.0", PublishedAt: 2018-12-10T00:59:57, Value: 0 }, { Name: "amqp", Version: "2.4.0", PublishedAt: 2019-01-13T13:25:57, Value: 3.0150071347031964 }, { Name: "redis", Version: "3.1.0", PublishedAt: 2019-01-29T00:22:34, Value: 0 }, { Name: "redis-py-cluster", Version: "1.3.6", PublishedAt: 2018-11-16T14:55:22, Value: 0 }, { Name: "kombu", Version: "4.2.2.post1", PublishedAt: 2019-01-01T14:00:05, Value: 0 }, { Name: "psycopg2", Version: "2.7.7", PublishedAt: 2019-01-22T17:55:35, Value: 0 }, { Name: "elasticsearch", Version: "6.3.1", PublishedAt: 2018-08-09T18:18:30, Value: 0 }, { Name: "tblib", Version: "1.3.2", PublishedAt: 2017-04-09T20:12:24, Value: 0 },  ], Total: 3.7082974061390157 } }
```

Comment:
The only change on this line was updating
```
{ Name: "kombu", Version: "4.2.1", PublishedAt: 2018-05-30T14:09:05, Value: 0 }
```
to
```
{ Name: "kombu", Version: "4.2.2.post1", PublishedAt: 2019-01-01T14:00:05, Value: 0 }
```

Based on https://pypi.org/project/kombu/#history, I think the old version was incorrect and this is indeed the correct version for 2019-02-01.